src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -1644,6 +1644,42 @@ namespace shogun
 			infer_backend(A, result)->scale(A, alpha, result);
 		}
 
+		/**
+		 * Scales every column of the matrix A with the respective coefficient
+		 * of alphas. i.e. result.column(i) = A.column(i) * alphas(i)
+		 * User should pass an appropriately pre-allocated memory matrix
+		 * Or pass the operands argument A as a result
+		 *
+		 * @param A First matrix
+		 * @param alpha Scale factor
+		 * @param result The scaled matrix
+		 */
+		template <typename T>
+		void scale(
+		    const SGMatrix<T>& A, SGMatrix<T>& result,
```

Comment:
no it doesnt match with this one here

Reply:
All the scale functions have differing signatures. The one in the `linalg` namespace is different from the one in the `linalgbackend`, so I just followed the same convention.

Reply:
i see, sorry about the noise then.